device_calendar/example/lib/main.dart

```diff
@@ -16,7 +16,7 @@ class _MyAppState extends State<MyApp> {
     return new MaterialApp(routes: {
       AppRoutes.calendars: (context) {
         return new CalendarsPage();
-      }
+      },     
```

Comment:
intentional trailing comma?

Reply:
I had code there but then removed it. I will remove the comma